source/_integrations/iotty.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: iotty
+description: "Instructions on how to connect iotty Smart Devices to Home Assistant."
+ha_release: "2024.8"
+ha_category: 
+  - Switch
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: no score
+ha_config_flow: true
+ha_codeowners:
+  - '@pburgio'
+ha_domain: iotty
+ha_platforms:
+  - switch
+ha_integration_type: hub
+---
+
+The iotty family is focused on beautiful aesthetics and includes the Smart Switch for lights and gates, the Smart Shades Switch for blinds and shutters and the Smart Outlet. Install the iotty devices and enjoy your new Smart Home with ease!
+
+
+  {% include integrations/config_flow.md %}
+
+## Supported devices
+
+This integration currently supports the following iotty Devices:
+
+- iotty Smart Switch \
+  US version
+  - https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389
+  - https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925
+  - https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269
+  - https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901 \
+  
+  EU VERSION:
+  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch
+  - https://iotty.uk/collections/frontpage/products/iotty-smart-switch?variant=40820222460082
+  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-luci-e-cancelli
+
+
+More integrations are coming soon:
+
+- iotty Shades Switch \
+  EU VERSION:
+  - https://iotty.uk/collections/frontpage/products/iotty-smart-shades-switch
+  - https://iotty.it/collections/frontpage/products/iotty-plus-interruttore-intelligente-per-tende-e-tapparelle
+
+- iotty Smart Outlet: \
+  EU VERSION:
+  - https://iotty.fr/collections/frontpage/products/prise-intelligent-iotty
+  - https://iotty.de/collections/frontpage/products/iotty-smart-steckdose
+  - https://iotty.it/collections/frontpage/products/iotty-plus-presa-intelligente
```

Comment:
Please add a section that gives more information about the supported entities. For example, Husquvarna or La Marzocco have sections on switches, for reference.